CMakeLists.txt

```diff
@@ -85,8 +85,11 @@ endif()
 option( VMTK_USE_SUPERBUILD
   "Build VMTK and the projects it depends on via SuperBuild.cmake." ON )
 
+set(VMTK_TEST_DATA_SOURCE "git-submodule" CACHE STRING "How to download test data (none, git-submodule)" FORCE)
+set_property(CACHE VMTK_TEST_DATA_SOURCE PROPERTY STRINGS none git-submodule)
```

Comment:
Sorry for being nitpicky, but it is probably better to clarify the details now than get back to it later.
What does "none" option mean? To me, source=none would suggest that no test data will be available (and so tests that require test data are skipped), but I don't think that the code does this (and then we could find a better name, such as `skip-data-with-tests`). If the actual behavior is to assume that the test data is already there then the option could be called something like "local" or "in-place".

Reply:
I'll change it to `in-place`, to me local sounded like it might imply copying the data from some local folder. I essentially want an option that cmake can assume the data will be there and shouldn't download/check it.